### DISCUSSION TITLE
Übersetzung "Probe"

### DISCUSSION THREAD

**Discussion by Sytec28** [rjcncpt/StarCitizen-Deutsch-INI#517] [Übersetzungsvorschläge]
Warum wurde dies nicht ins deutsche übersetzt?

Das deutsche Wort dafür wäre denke ich "Sonde", richtig?

**Comment by rjcncpt** [comment]
Hallo. Weil das tatsächlich in Missionstitel vorkommt. Das hatten wir anfangs übersetzt und das wurde als Fehler gemeldet.

Das wäre definitiv etwas was wir übersetzen würden aber da es im Titel vorkommt, geht das leider nicht. Hier im Screenshot siehst du das "Probe" zu den Missions Items zählt

![image](https://i.imgur.com/x13mUxN.png)

**Comment by Sytec28** [comment]
sorry, ich hatte die Regeln erst nacher gefunden!. Dann wurde es klar.
